platform/security/certificate/security-certificate-keystoreeditor/src/main/java/org/codice/ddf/security/certificate/keystore/editor/KeystoreEditorMBean.java

```diff
@@ -30,12 +30,86 @@
 
     List<Map<String, Object>> getTruststore();
 
-    void addPrivateKey(String alias, String keyPassword, String storePassword, String data, String type, String fileName)
+    void reInitializeKeystores();
```

Comment:
I don't think this is needed. This looks like a bug in the current implementation of this class to me. We should just always be reading from the keystores whenever an action is performed, rather than caching them.